dockerfiles/alpine/Dockerfile

```diff
@@ -75,9 +84,8 @@ RUN apk update \
   && adduser --shell /bin/bash -G bobd -G wheel -S bobd
 
 WORKDIR /home/bob/bin/
-COPY --from=cargo-build /usr/src/bob/target/x86_64-unknown-linux-musl/release/bobd .
-RUN chown bobd:bobd bobd \
-  && mkdir /bob/log -p && chown bobd:bobd /bob/log -R \
+COPY --chown=bobd:bobd --from=cargo-build /usr/src/bob/target/x86_64-unknown-linux-musl/${BUILD_PROFILE_DIR}/bobd .
```

Comment:
Here can be a problem: variable `BUILD_PROFILE_DIR` declared in other container, so there is a possibility, that it is not accessible here. This should be checked by building docker image with e.g. `test` profile